modules/dop/providers/projectpipeline/service.go

```diff
@@ -57,6 +57,7 @@ func (p *ProjectPipelineService) WithPermissionSvc(permission *permission.Permis
 type Service interface {
 	Create(ctx context.Context, params *pb.CreateProjectPipelineRequest) (*pb.CreateProjectPipelineResponse, error)
 	List(ctx context.Context, params deftype.ProjectPipelineList) ([]*dpb.PipelineDefinition, int64, error)
+	ListRef(ctx context.Context, params deftype.ProjectPipelineRefList) ([]string, error)
```

Comment:
I suggest add Ref -> UsedRef is more precise.

Reply:
done